### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello Guys,

I have created a Cart page for the E-commerce app.
I have also changed the card layout for the Wishlist page.

Please review and suggest.
Thanks

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
Looks good, go ahead with the merge

File: pages/wishlist.html [hunk 1]
```diff
@@ -67,125 +67,112 @@
...
         </div>
 
-        <div class="card-vertical">
+        <div class="card-vertical m-1">
           <div class="badge">
-            <img class="card-img-vertical" src="/images/cups.jpg" />
-            <span class="badge-text badge-on-left success">Bestseller</span>
+            <img class="card-img-vertical" src="/images/white-plate1.jpg" />
+            <span class="badge-text badge-on-left success px-0p5"
+              >Bestseller</span
```
Comment (samyukthasudhakar) [inline pages/wishlist.html:164]: Formatting can be taken care of 👆🏽

**Review by samyukthasudhakar — APPROVED** [review]
Love how neat and minimal the UI is. The code looks clean. Great work Apurva !

File: pages/wishlist.html (hunk 1 above)
Comment (ApurvaSawant11) [reply pages/wishlist.html:164]: Prettier is formatting it. When I format and save, it goes back to this.

**Review by ArjunJunna — APPROVED** [review]
Awesome work with the UI and code. Looks good to be merged.

**Pull request merged**